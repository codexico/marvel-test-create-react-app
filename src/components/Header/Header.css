.Header {
  margin-top: 35px;

  text-align: center;
}

.Header-title {
  margin: 17px 0 0;

  font-size: 32px;
}

.Header-subtitle {
  margin: 7px 0;

  font-size: 14px;
  font-weight: normal;
}
