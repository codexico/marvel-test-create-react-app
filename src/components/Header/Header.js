import React from 'react';

import { ReactComponent as Logo } from '../../img/logo.svg';

import './Header.css';

function Header() {
  return (
    <div className="Header">
      <Logo title="Marvel - Search heroes" />
      <h2 className="Header-title">EXPLORE O UNIVERSO</h2>
      <h3 className="Header-subtitle">
        Mergulhe no domínio deslumbrante de todos os personagens clássicos que
        você ama - e aqueles que você descobrirá em breve!
      </h3>
    </div>
  );
}

export default Header;
