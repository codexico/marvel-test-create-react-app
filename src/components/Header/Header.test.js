import React from 'react';
import { render } from '@testing-library/react';

import Header from './Header';

test('Component renders correctly', () => {
  const { asFragment } = render(<Header />);

  expect(asFragment()).toMatchSnapshot();
});
