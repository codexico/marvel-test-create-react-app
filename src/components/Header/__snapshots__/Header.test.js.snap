// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component renders correctly 1`] = `
<DocumentFragment>
  <div
    class="Header"
  >
    <svg
      title="Marvel - Search heroes"
    >
      logo.svg
    </svg>
    <h2
      class="Header-title"
    >
      EXPLORE O UNIVERSO
    </h2>
    <h3
      class="Header-subtitle"
    >
      Mergulhe no domínio deslumbrante de todos os personagens clássicos que você ama - e aqueles que você descobrirá em breve!
    </h3>
  </div>
</DocumentFragment>
`;
